app/src/main/java/org/p2p/wallet/auth/ui/pin/newcreate/NewCreatePinFragment.kt

```diff
@@ -61,6 +65,19 @@ class NewCreatePinFragment :
         requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
             presenter.onBackPressed()
         }
+
+        val deeplinkUri = deeplinksManager.pendingDeeplinkUri ?: return
+        if (DeeplinkUtils.hasFastOnboardingDeeplink(deeplinkUri)) {
```

Comment:
Let's move this to the presenter

Reply:
I think I will rewrite it today on splash screen logic

but I am against move URI to presenter

only maybe a method but it will be pure delegate :(